src/sagemaker/config/config_schema.py

```diff
@@ -882,11 +883,18 @@ def _simple_path(*args: str):
                     ADDITIONAL_PROPERTIES: False,
                     PROPERTIES: {
                         ADDITIONAL_INFERENCE_SPECIFICATIONS: {
```

Comment:
We'll remove this from the config with a comment, since this is not supported by the sdk

Reply:
Yes, removed the support for ADDITIONAL_INFERENCE_SPECIFICATIONS